platform-server-kernel/src/main/java/uno/cod/platform/server/core/dto/challenge/ChallengeCreateDto.java

```diff
@@ -10,6 +10,9 @@
     private String name;
 
     @NotNull
+    private Long organizationId;
+
```

Comment:
Then i think we should rename the attribute, because public is misleading. What about something that goes more in the direction like "shared" or "reusable" ...